docs/docs/api/nativeMethods/measure.md

```diff
@@ -46,13 +46,11 @@ const Comp = () => {
 
   useDerivedValue(() => {
     const measured = measure(aref);
-    // ...
+    if (measured !== null) {
+      // ...
+    }
   });
 
   return <View ref={aref} />;
 };
-```
-
-### Note
-
-You can use `measure()` only on rendered components. A good practice is to wrap the function call with a `try{} catch{}` block if there is a risk of calling the function on an item which is not rendered, for example: an invisible off screen item from FlatList.
```

Comment:
Can we bring this note back? The fact that it can be used only on already rendered components (e.g. `FlatList` items) is important.

Reply:
Done - I made a few changes, including updated wording (the old wording was no longer accurate), so let me know what you think.

Reply:
Sounds good, thanks!